html-aam/index.html

```diff
@@ -9365,6 +9365,132 @@ <h4 id="att-controls">`controls`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-command">`command`</h4>
+        <table aria-labelledby="att-command">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`command`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/popover.html#attr-command">`button`</a>;</td>
```

Comment:
was the support for this removed from "input" buttons ?

the html PR still mentions command / commandfor being allowed on input and button in the indices.html